lib/extensions/ar_migration.rb

```diff
@@ -1,6 +1,6 @@
 module ArPglogicalMigrationHelper
   def self.discover_schema_migrations_ran_class
-    return unless ActiveRecord::Base.connection.table_exists?("schema_migrations_ran")
```

Comment:
I chose to not fix this because although noisy, this the behavior we want in the future.  It will check only for table existence but not views.   We're checking for the table, not wanting a view.  We can change this to get rid of the warning but I'd rather live with this warning until we upgrade rails and get the new desired behavior.